debian/Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG JAVA_VERSION=17.0.7_7
+ARG DEBIAN_RELEASE=bullseye-20230522
```

Comment:
I (just now while reviewing) realize that this change as an impact: Dependabot won't update the version anymore (as it only supports the `FROM` instruction).

We have to switch to `updatecli` instead (as we already have some update manifests in the `updatecli/` directory, executed by a GitHub Action)

Reply:
Relates to https://github.com/jenkinsci/docker-ssh-agent/issues/242 .

Reply:
> I (just now while reviewing) realize that this change as an impact: Dependabot won't update the version anymore (as it only supports the `FROM` instruction).
> 
> We have to switch to `updatecli` instead (as we already have some update manifests in the `updatecli/` directory, executed by a GitHub Action)

How to track the Debian release? I haven't found (yet) a GitHub repo that would list the Debian releases.
Can `updatecli` parse an HTML file once retrieved via `http`?

Reply:
> How to track the Debian release?  I haven't found (yet) a GitHub repo that would list the Debian releases.
Can updatecli parse an HTML file once retrieved via http?

I guess you want to track the Docker image's version (like Dependabot does today). In that case, the https://www.updatecli.io/docs/plugins/resource/docker_image/ source should be enough.

Given the proposal from @ieugen, we should start by pinning the versions of both Bullseye and Bookworm in the PR here, and then do a subsequent PR with updatecli.